tests/test_common/test_vision/test_model/test_allconv.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
+import torch
+import torch.nn as nn
+
+from code_soup.common.vision.models.allconvnet import AllConv, AllConvNet

```

Comment:
This is inconsistent with our format, we do from code_soup.common.vision.models import AllConv, AllConvNet for which the classes should be a function from the package (aka it should be imported from models package.

You can check https://github.com/Adversarial-Deep-Learning/code-soup/blob/7f7d0e98adaab30672d492587b9da9f491d3ee7b/code_soup/ch5/models/__init__.py#L1

Similarily,
for tests, we need a hierarchy of tests later on so they should also be callable from packages, for reference 
https://github.com/Adversarial-Deep-Learning/code-soup/blob/7f7d0e98adaab30672d492587b9da9f491d3ee7b/tests/test_ch5/__init__.py#L1